zips/draft-stark-verify-nu.md

```diff
@@ -0,0 +1,156 @@
+ZIP: XXX
+Title: Circle STARK Verification as a Transparent Zcash Extension
+Owners: Abdel <abdel.dev.bitcoin@proton.me>
+Status: Draft
+Category: Consensus
+Created: 2025-10-14
+License: MIT
+Discussions-To: <https://forum.zcashcommunity.com/t/stark-verification-as-a-transparent-zcash-extension-tze-i-e-enabler-for-a-starknet-style-l2-on-zcash/52486>
+
+# Terminology
+
+The key words **MUST**, **MUST NOT**, **SHOULD**, **SHOULD NOT**, and **MAY** are to be interpreted as described in BCP 14 (RFC 2119 and RFC 8174).
```

Comment:
```suggestion
The key words "MUST", "SHOULD", and "MAY" are to be interpreted as described in BCP 14 [^BCP14] when, and only when, they appear in all capitals.
```
The convention is for this boilerplate to only mention keywords the document actually uses. Also remove the boldface formatting from keyword uses. (Boldface is used for them in the protocol spec but not in ZIPs.)